### ISSUE TITLE
v0.6.0-rc.5 examples/fn-args panics

### ISSUE THREAD

**Issue by thallada** [geofmureithi/apalis#398]
I know 0.6.0 is not released yet, but I was switching to it now to take advantage of the improved redis interface and cron backend but I noticed including `Data<TaskId>` in a job fns args causes a panic now.

I can reproduce it in the `examples/fn-args` project in this repo by running `cargo run`:

```
thread 'tokio-runtime-worker' panicked at /home/thallada/cloned/apalis/packages/apalis-core/src/service_fn.rs:47:34:
called `Option::unwrap()` on a `None` value
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

Sorry if this is a known issue and not fixed yet, I just wanted to post this in-case you weren't aware of it.

**Comment by geofmureithi** [comment]
Yeah this is a cron issue, it should be fixed ASAP.
To work around, you can add `data` entry.

**Comment by geofmureithi** [comment]
Out of curiosity, what version of 0.6 are you using?

**Comment by geofmureithi** [comment]
Should be fixed by #409
